app/vendor/MathJax/config.js

```diff
@@ -6,7 +6,7 @@ var MathJax = {
     ignoreClass: 'nostem|nolatexmath'
   },
   asciimath2jax: {
-    delimiters: [['\\$', '\\$']],
+    delimiters: [['\\$', '\\$'], ['\\$\\$', '\\$\\$']],
```

Comment:
@mojavelinux What do you think ? Should we also add this delimiter and the two hooks below in the default HTML5 converter ?

Reply:
I needed to introduce the new delimiters because in `app/vendor/MathJax/config/TeX-MML-AM_HTMLorMML.js` I need to target all the asciimath blocks and leave the asciimath inlines untouched.

Previously to that I targeted the existing $...$ delimiters, however I had the result of having both inline and block affected (centered, clear both).